optuna/integration/allennlp.py

```diff
@@ -57,6 +57,13 @@ class AllenNLPExecutor(object):
     `config file <https://github.com/optuna/optuna/blob/master/
     examples/allennlp/classifier.jsonnet>`_.
 
+    .. note::
+        In :class:`~optuna.integration.AllenNLPExecutor`,
+        you can pass parameter to AllenNLP by either defining a search space using
+        Optuna suggest methods or setting a environment variable just like AllenNLP cli.
```

Comment:
A nit. 
- I propose a plural form to show that we can use multiple parameters clearly.
- I usually capitalize `cli` because it's an abbreviation. If you have specific reasons, please ignore it.

```suggestion
        you can pass parameters to AllenNLP by either defining a search space using
        Optuna suggest methods or setting environment variables just like AllenNLP CLI.
```